### PR TITLE
Sets up dev environment

### PR DESCRIPTION
**Pull request by devinlizardi**
setting up development environment with `docker compose` 

updated README

### CODE REVIEW

File: compose.yaml
Comment (mrodz) [inline compose.yaml:1]: My only observation is that if the `PORT` environment variable is set to a value other than `5000`, then the container will not expose the correct port.

**Comment by mrodz** [comment]
This looks good to me. Good explanation of the build/deploy process. It appears that the docker compose works as well.

**Review by mrodz — APPROVED** [review]
Running `docker compose up` spins up a server that loads an IncrediBuild sign in page. I assume this is the expected behavior! Looks good to me.

**Comment by devinlizardi** [comment]
> Running `docker compose up` spins up a server that loads an IncrediBuild sign in page. I assume this is the expected behavior! Looks good to me.

that's very strange! do you mind attaching a screenshot of what you're seeing? it should look something like this, which I'm getting from the [prod URL](https://www.api.latale.org/getRanking)

latale-api

**Comment by devinlizardi** [comment]
@mrodz realized you may need the `.env` file to see anything! my bad I should've provided it

ping me on Discord when you see this comment I'll send it there

**Pull request merged**